src/themes/custom/app/request-copy/deny-request-copy/deny-request-copy.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component } from '@angular/core';
+import {
+    DenyRequestCopyComponent as BaseComponent
+} from 'src/app/request-copy/deny-request-copy/deny-request-copy.component';
+
+@Component({
+    selector: 'ds-deny-request-copy',
+    // styleUrls: ['./deny-request-copy.component.scss'],
+    styleUrls: [],
+    // templateUrl: './deny-request-copy.component.html',
+    templateUrl: './../../request-copy/deny-request-copy/deny-request-copy.component.html',
```

Comment:
This path also looks wrong...it should point at the HTML file under /src/app/, but it never even makes it up to the /src/app folder.

Reply:
Yes, this is wrong:  it should be three more levels up, then down through `app/`.  I must've been looking at the wrong example when I did this.